apps/trading/components/rewards-container/simple-reward-card.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Button,
+  Intent,
+  Pill,
+  VegaIcon,
+  VegaIconNames,
+} from '@vegaprotocol/ui-toolkit';
+import { useT } from '../../lib/use-t';
+import ReactMarkdown from 'react-markdown';
+import { Link } from 'react-router-dom';
+import { GradientText } from '../gradient-text';
+import { ColourfulBorder } from 'libs/ui-toolkit/src/utils/border';
+
+type SimpleRewardCardProps = {
+  title: string;
+  description: string;
+  tags?: string[];
+  link?: string;
+};
+export const SimpleRewardCard = ({
+  title,
+  description,
+  tags,
+  link,
+}: SimpleRewardCardProps) => {
+  const t = useT();
+  return (
+    <div className="rounded relative p-px">
+      <ColourfulBorder className="!rounded" />
```

Comment:
I don't think `!rounded` is required. I believe the `cn` function should allow you to pass in a class, and if there is a conflicting class it will replace it using twMerge.

Can you check